apps/mobile/src/libs/components/typing-text-view/libs/hooks/use-typing-text/use-typing-text.hook.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from "~/libs/hooks/hooks";
+
+type UseTypingTextArguments = {
+	content: string;
+	firstLetterIndex: number;
+	lettersPrintedPerStep: number;
+	nextLettersPrintedDelay: number;
+	startTypingDelay: number;
+};
+
+type UseTypingTextReturnData = {
+	hiddenLetters: string;
+	visibleLetters: string;
```

Comment:
```suggestion
	hiddenContent: string;
	visibleContent: string;
```
Only arrays should have a plural naming